test/acceptance/tests/validation/validation_test.go

```diff
@@ -823,8 +824,7 @@ func TestValidation_MeshGateway(t *testing.T) {
 			kind:      "mesh-gateway",
 			lbEnabled: true,
 			lbSubnets: []string{"subnet"},
-			lbVpcID:   "vpc",
-		},
+			lbVpcID:   "vpc"},
```

Comment:
```suggestion
			lbVpcID:   "vpc"
		},
```

nit: minor indentation